test/bin/test

```diff
@@ -12,20 +12,17 @@
 # Configuration
```

Comment:
Changes done here are in conflicts with those done there.

https://github.com/FriendsOfSymfony1/symfony1/pull/270/files#diff-1d56fbd8b67d0ff6b08b293bba30e27816f34e8815d901685e133972646828a3